roles/result/templates/result.json

```diff
@@ -1,7 +1,7 @@
 {
   "uuid": "{{ item.item | regex_replace('-.*','') | to_uuid }}",
   "user": "{{user}}",
-  "spec": {{  _benchmark_example_com_benchmark.spec | to_nice_json  }},
+  "spec": {{  _ripsaw.cloudbulldozer.io_benchmark.spec | to_nice_json  }},
```

Comment:
I think this should be updated to `_ripsaw_cloudbulldozer_io_benchmark` as that's how ansible-operator sends the extra vars, more on this is here https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/dev/developer_guide.md#extra-vars-sent-to-ansible